tables/endpointsecurity/src/fileeventstableplugin.cpp

```diff
@@ -0,0 +1,171 @@
+#include "fileeventstableplugin.h"
+
+#include <chrono>
+#include <mutex>
+
+
+namespace zeek {
+struct FileEventsTablePlugin::PrivateData final {
+  PrivateData(IZeekConfiguration &configuration_, IZeekLogger &logger_)
+      : configuration(configuration_), logger(logger_) {}
+
+  IZeekConfiguration &configuration;
```

Comment:
This seems unused.

Reply:
This is used in the constructor to get `maxQueuedRowCount`.